### ISSUE TITLE
Projektideen von (Elektro-)Werkzeugseiten verlinken

### ISSUE THREAD

**Issue by LukasFreeze** [comakingspace/do-something#109]
Gerade bei den größeren/"professionelleren" Geräten fragen Neulinge oft, was man damit denn überhaupt machen kann. Ich fänd es cool, wenn wir es schaffen würden, von jeder Wikiseite eines Elektrogeräts oder einer "Maschine" **zwei** Projektseiten zu verlinken, auf denen diesem Gerät eine prominente Rolle zukommt! Wenn es noch nicht genug Projekte gibt und/oder jemand einfach nur ein Foto parat hat, ist das natürlich auch hilfreich.

Dafür ist die "What Links Here"-Funktion im Wiki sehr nützlich, ich habe es mal mit der Dekupiersäge vorgemacht:
**1)**
![grafik](https://user-images.githubusercontent.com/19755883/55569662-4030c580-5702-11e9-9c7d-38333b7d2cd5.png)
**2)**
![grafik](https://user-images.githubusercontent.com/19755883/55569736-60f91b00-5702-11e9-9a19-913c333c563a.png)

Zum Verlinken würde ich wie üblich eine kleine ["Navigation Gallery"](https://wiki.comakingspace.de/Project:Navigation_Galleries) vorschlagen ;-)

![grafik](https://user-images.githubusercontent.com/19755883/55570406-acf88f80-5703-11e9-8a5a-444bc655629e.png)

(related: #96 )

**Comment by NitramLegov** [comment]
Schaffliste, bitte jeweils updaten (gerne auch mit "not applicable", falls ein Gerät nichts braucht):

| Tool | Status |
|---|---|
|Scroll Saw| done |
|Impact Drills| not applicable |
|Disc Grinders| |
|Bench Grinders| |
|Belt Sander| |
|Chop Saw| |
|Circular Saws| |
|Cordless Drills| not applicable |
|Drill Press| not applicable |
|Rotary Tool| |
|Eccentric Sanders| |
|Heat Gun| not applicable |
|Hot Air Soldering Station| |
|Hot Glue Gun| |
|Jigsaws| |
|Labelmaker| not applicable |
|Pyrography Burner| |
|Soldering Irons| not applicable |
|Shop Vacs| not applicable |
|Corded Drills| not applicable |
|Rotary Hammer| not applicable |
|Welding Transformer| |
|Bandsaws| |
|Router Table| |
|Oscillating Tool| |
|Reciprocating Saw| |
|Biscuit Joiner| |
|Sheet Sanders| not applicable |
|CNC Router| |
|Metal Lathes| |
|Projector| not applicable |
|Wood Lathes| |
|Blast Cabinet| |
|Prusa i3 MK2S| |
|Fridge| not applicable |
|Plastic Shredder| |

**Comment by LukasFreeze** [comment]
Danke für die Liste!

Ich hab mir mal erlaubt, die "not applicables" etwas zu ändern - oder hatte es einen Grund, dass zB die Schwingschleifer nichts bekommen sollten aber die Exzenterschleifer schon? So oder so sind manche Werkzeuge natürlich selten der Mittelpunkt eines Projekts, aber ich fänds trotzdem cool wenn wir auf lange Sicht für blutige Anfänger ein paar Anwendungsfotos hinzufügen würden :-)

**Comment by NitramLegov** [comment]
Wenn wir eine "Made With::" Property einfügen können wir das auch über ein Template mit semantic query lösen und müssten nur die Projektseiten pflegen :-)

**Comment by LukasFreeze** [comment]
Ja, das müsste auch "andersrum" funktionieren! Allerdings finde ich, dass nicht jedes Projekt mit Verwendung einer best. Maschine so prominent auf der Maschinenseite auftauchen sollte, sondern eben nur je 2-3 besonders gute Beispiele. Die Kurationsarbeit wäre also im Wesentlichen die gleiche, nur dass wir mit der manuell gebauten Galerie den Stil und die captions komplett im Griff hätten 🤔
